Modules/Control/Field_To_Current.py

```diff
@@ -0,0 +1,17 @@
+import math
+import numpy
+AMOUNT_OF_WRAPS = 100
+RADIUS = 0.09 #(meter)
+RESISTANCE = 170 #(ohm)
```

Comment:
You should move all of the constant to a seprate config file.
There will probably be more constants along the way and it will be a pain to find them if the needed to be changed.